.github/workflows/ci.yml

```diff
@@ -258,11 +258,22 @@ jobs:
         run: npm test &> /dev/null || exit 0
 
       - name: Accept baselines
-        run: npx hereby baseline-accept
+        run: |
+          npx hereby baseline-accept
+          git add tests/baselines/reference
 
-      - name: Check for unused baselines
+      - name: Check baselines
         run: |
-          if ! git diff --exit-code --quiet; then
-            echo "Unused baselines:"
-            git diff --exit-code --name-only
+          function print_diff() {
+            if ! git diff --staged --exit-code --quiet --diff-filter=$1; then
+              echo "$2:"
+              git diff --staged --name-only --diff-filter=$1
```

Comment:
Are all `diff-filter`s single word? With the current use of `print_diff` there's no need to quote the expansion, but do you think it would make sense to quote it to avoid possible issues in the future?

Reply:
They're restricted to only the single letters I have written here, concatenated together. This should be plenty future proof as this is a git feature.